### PR TITLE
[examples:todos-app]: Add vite

### PR DESCRIPTION
**Pull request by shinshin86**
In `@sveltejs/kit` there was a BREAKING CHANGE as follows.
Currently todos-app is not working because of this change, so i have a fix it.
https://github.com/sveltejs/kit/pull/5332

### CODE REVIEW

File: examples/todos-app/src/styles/tailwind-output.css [hunk 1]
```diff
@@ -433,11 +433,6 @@ input::-moz-placeholder, textarea::-moz-placeholder {
   color: #9ca3af;
 }
 
-input:-ms-input-placeholder, textarea:-ms-input-placeholder {
```
Comment (shinshin86) [inline examples/todos-app/src/styles/tailwind-output.css:436]: I have noticed that diff are being generated so I am pushing the fixes together.

**Review by FabianLars — APPROVED** [review]
thanks a bunch!

**Comment by FabianLars** [comment]
aahhh, almost. We require _all_ commits to be signed. So if you could sign your commits, i can merge it right away ❤️ 

p.s. here's one possible guide https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Comment by shinshin86** [comment]
@FabianLars Thank you!

I have set up a signature for the commit.
I am pushing an empty commit with signed.
Is this alright with you?

**Comment by FabianLars** [comment]
_All_ commits need to be signed. i literally can not click on "Merge pull request" otherwise (blocked by github settings), so we'll need a force-push from you to overwrite the old commits :/

**Comment by shinshin86** [comment]
@FabianLars OK!
I committed again with all commits signed and then force pushed.

**Pull request merged**